packages/expo-updates/ios/EXUpdates/AppLoader/StringStringDictionarySerializer.swift

```diff
@@ -0,0 +1,70 @@
+//  Copyright © 2021 650 Industries. All rights reserved.
+
+import Foundation
+
+public enum SerializerError: Error {
```

Comment:
I'm honestly surprised that there are not built-in Swift methods for some of these